src/app/migrations/migration.coffee

```diff
@@ -0,0 +1,37 @@
+async = require 'async'
+semver = require 'semver'
+log = require('../lib/persistent_log')
+    prefix: "Filter Manager"
+    date: true
+
+
+# corresponding of migration file
+migrations = [
+    '1.2.0'
+    '1.1.1'
+    '1.0.0'
+    '0.3.0'
+]
+
+
+module.exports =
+
+
+    migrate: (oldVersion, callback) ->
+        log.debug 'migrate'
+
+        migrationsSorted = migrations.sort()
```

Comment:
For the record, the sort doesn't respect semver: 1.10.0 will come before 1.2.0 for example.